src/badges/tests/test_models.py

```diff
@@ -19,6 +19,15 @@ def setUp(self):
     def test_badge_rarity_creation(self):
         self.assertIsInstance(self.common, BadgeRarity)
         self.assertEqual(str(self.common), self.common.name)
+    
+    def test_default_badge_rarities_created(self):
```

Comment:
This test should be moved to the app that has the code being tested.  Can you move this to a new `test_initialization.py` in the tenant app?

Reply:
Also the comment here is wrong. this isn't a "migration" (that means a specific thing in databases and django).  Comment should be something like:
"""" When a new tenant is created, the initialization script should create 6 default BadgeRarity objects.""""

However, you'll need to make a new test class in the new test_initialization.py, and make sure it inherits from `TenantTestCase` (like almost all other test class in the project)